src/rmp/README.md

```diff
@@ -1,6 +1,7 @@
-### Restructure
+# Restructure
+
 **Restructure** is an interface to ABC for re-synthesis. The package allows
-re-structuring targetting area or timing. It extracts cloud of logic using
+re-structuring targeting area or timing. It extracts cloud of logic using
```

Comment:
restructuring targeting -> change to logic restructuring,

Reply:
Fixed on f028b29